Yoast/wordpress-seo
# Standardize multisite admin screens and admin bar

## Summary

This PR can be summarized in the following changelog entry:

* Standardize the multisite settings screen.

Once this is merged, the sub-issues that went into this must be considered as well (include each of their changelog entries and set the milestone accordingly):

* #9690 
* #9871 
* #9944 
* #9976 

## Relevant technical choices:

* See the respective sub-issues. All changes were already reviewed in one of those issues.

## Test instructions

This PR can be tested by following these steps:

* Before testing: All of those things have already been acceptance-tested through the sub-issues, so you don't need to be very thorough, as nothing was actually changed _in_ this PR. It's just an extra round to verify the following things work correctly.
* Set up a multisite, and activate the plugin network-wide.
* Go to the network admin.
* Go to the plugin's network settings screen.
* Change settings around in different ways, save them and verify they are persisted correctly.
* Go to the site admin.
* Go to the plugin's regular settings screen.
* Change some settings away from their defaults and save them.
* Go back to the network admin and the plugin's network settings screen.
* Restore the settings for the site where you just changed them.
* Go back to that site's plugin settings screen and ensure the settings are correctly set back to the defaults.
* Go to the network admin.
* Verify that the admin bar contains network-specific content, i.e. links to the network settings screen and sub-pages. It should not contain a notification bubble at the network level, since notifications are only relevant per site.

## Quality assurance


Fixes #9447
